extern/agg-2.4/include/agg_scanline_u.h

```diff
@@ -458,7 +458,7 @@ namespace agg
     class scanline32_u8_am : public scanline32_u8
     {
     public:
-        typedef scanline_u8           base_type;
+        typedef scanline32_u8         base_type;
```

Comment:
@djkirkham Great spot! 

The `typedef` for the `base_type` should match the base class of this derived class.

And your change also matches up with the matplotlib [agg implementation](https://github.com/matplotlib/matplotlib/blob/master/extern/agg24-svn/include/agg_scanline_u.h#L461).